### PR TITLE
italian translation added

### PR DESCRIPTION
**Pull request by legacychimera247**
Helloooooo

just added the italian translation :)

### CODE REVIEW

**Review by Olf0 — COMMENTED** [review]
@legacychimera247, thank you very much for contributing an Italian translation.

May I ask, why you preferred to directly pose a pull-request instead of using Transifex, as suggested in [the translations README](https://github.com/sailfishos-chum/sailfishos-chum-gui/tree/main/translations#readme)?

#### Background
Though I am not at all a fan of Atlassian (the company behind Transifex), Transifex eases collaborating with other translators, provides translating-specific tooling (e.g. its "translation memory" and its GitHub-integration makes my life as co-maintainer of the SailfishOS:Chum GUI application easier.
The biggest issue is getting translations directly contributed via a PR up to Transifex: Sometimes they are automatically picked up, but most often they have to be manually transferred, which is a tedious and cumbersome process.

#### Please provide feedback
Hence, if you do not have strong reasons not to use Transifex, please continue maintaining this translation there, as soon as I merge this PR and took care that it arrives well at Transifex.
If you do have strong reasons not to use Transifex, please name them here.
<br />

Cheers & thanks!

**Comment by legacychimera247** [comment]
i have to be honest, i couldn't wait to have this translated and doing it directly was the fastest way, as i went into transifex and had to request a language (which i did anyway)

i have (almost) no problem in using transifex, and i do suppose i can use that next time (i guess now transifex should pick my file right?)

(althought i hate the fact you have to request things on there, it could have been simpler like weblate, but anyway,...)

**Comment by Olf0** [comment]
> i have to be honest, i couldn't wait to have this translated and doing it directly was the fastest way,

Sorry to say that this bluntly, but either way my little available time will be the limiting factor.

> as i went into transifex and had to request a language (which i did anyway)

Thank you for denoting that, Transifex regularly misses to send notifications:  I wil take care about that.

> i have (almost) no problem in using transifex, and i do suppose i can use that next time (i guess now transifex should pick my file right?)

This is exactly the issue: Most of the time Transifex fails to pick up new translations when they first appear in the target repository.  I often ended up tediously importing them manually in Transifex.

> (althought i hate the fact you have to request things on there, it could have been simpler like weblate, but anyway,...)

Weblate was not available when most SailfishOS apps started, unfortunately.  I hate Atlassian, it is a bad, data-gathering, proprietary software company, with a horrible security record.  All our translations are fed into Atlassian's translation database to be used for their commercial services.

**Comment by legacychimera247** [comment]
ok, let's see what happens and if translation is not taken i will do it again then...

**Comment by Olf0** [comment]
I had to deny your request at Transifex, because you requested the "wrong" language: For details and reason, please see commit f9ac457.

The synchronisation from GitHub to Transifex failed, as usual and expected.  Hence I created `it` manually.  I will manually upload the translation there this weekend.

Furthermore, I cannot add you as translator to `it` at Transifex, because your profile there ("luca 247" / 247) is either private ("invisible") or deleted.

**Comment by Olf0** [comment]
> I will notify you here.

Done.

I had to input the plural forms `%n aggiornamento disponibile` / `%n aggiornamenti disponibili` / `%n aggiornamenti disponibili` again for some reason.  I will request a review from you to check if this is correct, when Transifex pushes this change to this target repository.

**Comment by Olf0** [comment]
@legacychimera247, please review https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/308/files
and denote your opinion in a comment here: https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/308

**Comment by legacychimera247** [comment]
hello, sorry if long time has passed, kinda got bored of translating back then, but i'm still on my mission to translate the sailfish ecosystem

i'll have a look at the changes made on chum, and also will try to address the issues you pointed out here (limited to my capabilities ofc)

**Comment by legacychimera247** [comment]
too short,  did read

i did not take any criticism, don't you worry, probably it's just that i have translated roughly 30 apps on sf + some on desktop and i kinda needed a break

if you want to have a laugh, i was using the plex client for desktop and said to me "shame it's already translated, i could have done i. well let me see in credit who did it"

me

i do even forget things i do :)

anyway i don't mind being added as a storeman translator, nice idea indeed, as despite all, i really want sailfish to be as localized as possible

i'll have a look at storeman anyway althought i didn't really understood yet how to use transifex

it seems to me that i have to have a specific link for the app, cause if i enter transifex i only see jock between projects i translate...

**Comment by legacychimera247** [comment]
it's 100% translated as i found the correct link and finished it already :) but this morning i wanted to change a (very old) wrong translation and spent about 15 minutes to find how to do it

oh well, anyway now app is 100% done :) no point in reviewing those as i did it myself, i would confirm my version :P

**Comment by Olf0** [comment]
> * If you want me to add you as a `it` translator at Transifex for any other app I maintain (e.g. Daily Comics, FlowPlayer, FileCase) or co-maintain (e.g. MeeCast, Patchmanager), please let me know.

I took a look at "my" apps (I wrote none of them), detecting that FileCase's and FlowPlayer's Italian translations are almost complete (~ 95%), except for one or two strings I altered since I started maintaining them.  Hence I assigned you as "translator" for these two apps at Transifex.  Feel free to translate the untranslated strings (i.e. the primary task), review the extant strings (i.e. the secondary task) and install & use these apps (tertiary task), as your time and motivation allows for.  The only thing I ask for: Please do denote if you pause on of these tasks after you have one reached "100% strings translated": As you know that Transifex automatically poses a PR to this repo then, please message me there if you take a break, so I won't review that PR.

TL;DR
Next:
- https://app.transifex.com/olf/filecase/language/it/
- https://app.transifex.com/olf/flowplayer-1/language/it/

**Comment by Olf0** [comment]
Thank you very much for the PRs **https://github.com/sailfishos-applications/filecase/pull/89** and **https://github.com/sailfishos-applications/flowplayer/pull/113**.
<sub>I will likely merge them on Sunday.<sub>

Nice, two more stale translations revived!

**Comment by legacychimera247** [comment]
i saw someone also made me translator for daily comic, not right now but i will do that for sure :)
